pkg/controller/containerjfr/containerjfr_controller.go

```diff
@@ -96,7 +96,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	// TODO(user): Modify this to be the types you create that are owned by the primary resource
 	// Watch for changes to secondary resource Pods and requeue the owner ContainerJFR
```

Comment:
Could you change this comment to read Deployments instead of Pods, and remove the TODO above it?